zebra/zebra_mpls.c

```diff
@@ -2617,6 +2654,42 @@ int mpls_ftn_update(int add, struct zebra_vrf *zvrf, enum lsp_types_t type,
 	return 0;
 }
 
+int mpls_ftn_uninstall(struct zebra_vrf *zvrf, enum lsp_types_t type,
```

Comment:
Add comment to precise the purpose of this new function

Reply:
This function is already documented in the header file (_zebra/zebra_mpls.h_).